spk/jackett/Makefile

```diff
@@ -1,19 +1,17 @@
 SPK_NAME = jackett
-SPK_VERS = 0.16.1340
+SPK_VERS = 0.17.474
 SPK_REV = 12
 SPK_ICON = src/jackett.png
 DSM_UI_DIR = app
 
-DEPENDS = cross/busybox cross/curl cross/$(SPK_NAME)
-# depend on mono 5.4 for BTLS HttpWebClient support
-SPK_DEPENDS = "mono>=5.4"
+DEPENDS = cross/$(SPK_NAME)
 
-MAINTAINER = m4tt075
+MAINTAINER = SynoCommunity
 DESCRIPTION = "Jackett works as a proxy server. It translates queries from apps into tracker-site-specific http queries, parses the html response, then sends results back to the requesting software. Jackett is a single repository of maintained indexer scraping and translation logic - removing the burden from other apps."
 RELOAD_UI = yes
 DISPLAY_NAME = Jackett
 STARTABLE = yes
-CHANGELOG = "Add workaround for mono bug with armv5/88f6281"
+CHANGELOG = "Compile using .NET Core"
```

Comment:
I would manage ChangeLog conditionally according to target arch... either mono or dotnetcore